code/game/objects/items/weapons/disks.dm

```diff
@@ -5,3 +5,78 @@
 	icon_state = "datadisk0"
 	drop_sound = 'sound/items/handling/disk_drop.ogg'
 	pickup_sound =  'sound/items/handling/disk_pickup.ogg'
+
+/obj/item/disk/data
+	name = "Cloning Data Disk"
+	icon_state = "datadisk0" //Gosh I hope syndies don't mistake them for the nuke disk.
+	var/datum/dna2/record/buf = null
+	var/read_only = FALSE //Well,it's still a floppy disk
+
+/obj/item/disk/data/proc/initialize()
```

Comment:
I just realized this is not `Initialize()`

is there any particular reason for this

Reply:
This proc isn't even run I think, how was this tested?

Reply:
I just moved existing code which was in cloning.dm IIRC to here